frontend/resourceadm/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactNode } from 'react';
+import classes from './Modal.module.css';
+import ReactModal from 'react-modal';
```

Comment:
vet ikke hvordan det er i studio men i autorisasjon og apps er veldig konservative på bruk av tredjepartspakker for det kan få ganske store følger dersom den plutselig slutter å bli supportet osv. Så vi prøver å lage komponentene vi trenger fra bunn av. 

Det kan hende det er noen av de andre teamene som har laget en modal-komponent fra bunn som det går an å bruke. Men ville i alle fall klarert det med noen i studio at det er greit å legge til den komponenten.

Reply:
Takk! Det visste jeg ikke om. Har laget en issue på det nå #10580. Oppdaterer den når jeg ikke har andre oppgaver :)

Reply:
> Men ville i alle fall klarert det med noen i studio at det er greit å legge til den komponenten.

Mente egentlig høre med studio om det er greit å legge til det tredjepartsbiblioteket

Reply:
Hører med de jeg, takk :)